2022-05-28 17:17:23,507 INFO    MainThread:261984 [wandb_setup.py:_flush():76] Loading settings from /home/cidar/.config/wandb/settings
2022-05-28 17:17:23,507 INFO    MainThread:261984 [wandb_setup.py:_flush():76] Loading settings from /home/cidar/Desktop/refactor2/wandb/settings
2022-05-28 17:17:23,507 INFO    MainThread:261984 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-05-28 17:17:23,507 INFO    MainThread:261984 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train_patch_gan.py', 'program': '/home/cidar/Desktop/refactor2/train_patch_gan.py'}
2022-05-28 17:17:23,507 INFO    MainThread:261984 [wandb_init.py:_log_setup():428] Logging user logs to /home/cidar/Desktop/refactor2/wandb/run-20220528_171723-1ctfulrs/logs/debug.log
2022-05-28 17:17:23,508 INFO    MainThread:261984 [wandb_init.py:_log_setup():429] Logging internal logs to /home/cidar/Desktop/refactor2/wandb/run-20220528_171723-1ctfulrs/logs/debug-internal.log
2022-05-28 17:17:23,508 INFO    MainThread:261984 [wandb_init.py:init():462] calling init triggers
2022-05-28 17:17:23,508 INFO    MainThread:261984 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {'config': 'yaml/patch_gan.yaml', 'model_name': 'patch_gan', 'exp_name': 'patch_gan_test', 'dataset_size': 'resolution_dataset50', 'dataset_name': 'z_axis', 'num_epochs': 10, 'num_workers': 8, 'seed': 123, 'lr': 0.0001, 'n_freq': 2, 'batch_size': 32, 'train_batch_size': 32, 'val_batch_size': 1, 'factor': 2, 'wandb': True, 'project_name': 'super_resolution', 'n_blocks': 4, 'start_filters': 32, 'activation': 'relu', 'normalization': 'batch', 'conv_mode': 'same', 'dim': 2, 'up_mode': 'transposed', 'init': 'kaiming', 'weights_file': 'None', 'patch': False, 'tensorboard': False, 'gan_mode': 'lsgan', 'lr_G': 0.0001, 'lr_D': 0.0001, 'lambda_L1': 100, 'n_down': 3, 'num_filters': 64, 'device': device(type='cuda'), 'val_image_dir': 'resolution_dataset50/z_axis/factor_2/val', 'val_label_dir': 'resolution_dataset50/z_axis/label/val', 'train_image_dir': 'resolution_dataset50/z_axis/factor_2/train', 'train_label_dir': 'resolution_dataset50/z_axis/label/train', 'epoch_image_path': 'resolution_dataset50/z_axis/factor_2/train/lr_f1_160_2_z_46.png', 'checkpoints_dir': 'outputs/resolution_dataset50/patch_gan/patch_gan_test/checkpoints/z_axis/factor_2/', 'epoch_images_dir': 'outputs/resolution_dataset50/patch_gan/patch_gan_test/epoch_images/z_axis/factor_2/', 'input_images_dir': 'outputs/resolution_dataset50/patch_gan/patch_gan_test/input_images/z_axis/factor_2/', 'output_images_dir': 'outputs/resolution_dataset50/patch_gan/patch_gan_test/output_images/z_axis/factor_2/', 'loss_dir': 'outputs/resolution_dataset50/patch_gan/patch_gan_test/losses/z_axis/factor_2/', 'grad_norm_dir': 'outputs/resolution_dataset50/patch_gan/patch_gan_test/grad_norm/z_axis/factor_2/', 'plot_dir': 'outputs/resolution_dataset50/patch_gan/patch_gan_test/plots/z_axis/factor_2/'}
2022-05-28 17:17:23,508 INFO    MainThread:261984 [wandb_init.py:init():515] starting backend
2022-05-28 17:17:23,508 INFO    MainThread:261984 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-05-28 17:17:23,532 INFO    MainThread:261984 [backend.py:ensure_launched():217] starting backend process...
2022-05-28 17:17:23,555 INFO    MainThread:261984 [backend.py:ensure_launched():222] started backend process with pid: 262114
2022-05-28 17:17:23,556 INFO    MainThread:261984 [wandb_init.py:init():525] backend started and connected
2022-05-28 17:17:23,559 INFO    MainThread:261984 [wandb_init.py:init():596] updated telemetry
2022-05-28 17:17:23,560 INFO    MainThread:261984 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2022-05-28 17:17:24,151 INFO    MainThread:261984 [wandb_run.py:_on_init():1924] communicating current version
2022-05-28 17:17:24,220 INFO    MainThread:261984 [wandb_run.py:_on_init():1928] got version response 
2022-05-28 17:17:24,220 INFO    MainThread:261984 [wandb_init.py:init():662] starting run threads in backend
2022-05-28 17:17:25,997 INFO    MainThread:261984 [wandb_run.py:_console_start():1898] atexit reg
2022-05-28 17:17:25,998 INFO    MainThread:261984 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.REDIRECT
2022-05-28 17:17:25,998 INFO    MainThread:261984 [wandb_run.py:_redirect():1776] Redirecting console.
2022-05-28 17:17:25,999 INFO    MainThread:261984 [wandb_run.py:_redirect():1832] Redirects installed.
2022-05-28 17:17:26,000 INFO    MainThread:261984 [wandb_init.py:init():688] run started, returning control to user process
2022-05-28 17:17:26,000 INFO    MainThread:261984 [wandb_watch.py:watch():47] Watching
2022-05-28 17:18:32,126 INFO    MainThread:261984 [wandb_run.py:_finish():1685] finishing run super_resolution/super_resolution/1ctfulrs
2022-05-28 17:18:32,126 INFO    MainThread:261984 [wandb_run.py:_atexit_cleanup():1867] got exitcode: 0
2022-05-28 17:18:32,127 INFO    MainThread:261984 [wandb_run.py:_restore():1839] restore
2022-05-28 17:18:34,665 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 569
  total_bytes: 569
}

2022-05-28 17:18:34,771 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 569
  total_bytes: 569
}

2022-05-28 17:18:34,958 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 569
  total_bytes: 569
}

2022-05-28 17:18:35,199 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 569
  total_bytes: 23618
}

2022-05-28 17:18:35,300 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 569
  total_bytes: 289817
}

2022-05-28 17:18:35,401 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 289817
  total_bytes: 289817
}

2022-05-28 17:18:35,502 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 289817
  total_bytes: 289817
}

2022-05-28 17:18:35,605 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 289817
  total_bytes: 289817
}

2022-05-28 17:18:35,707 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 289817
  total_bytes: 289817
}

2022-05-28 17:18:36,353 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 289817
  total_bytes: 289817
}

2022-05-28 17:18:36,611 INFO    MainThread:261984 [wandb_run.py:_on_finish():1996] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 289817
  total_bytes: 289817
}
local_info {
}

2022-05-28 17:18:37,729 INFO    MainThread:261984 [wandb_run.py:_footer_history_summary_info():3104] rendering history
2022-05-28 17:18:37,737 INFO    MainThread:261984 [wandb_run.py:_footer_history_summary_info():3136] rendering summary
2022-05-28 17:18:37,739 INFO    MainThread:261984 [wandb_run.py:_footer_sync_info():3060] logging synced files
